Pipfile

```
@@ -7,11 +7,10 @@ name = "pypi"
 #
 # NuCypher
 #
+umbral = "*"
```

Do we want to pin a version specifier here?

```suggestion
umbral = "=>0.1.0a.3"
```

---

It was under my impression that we wanted NuCypher to fail fast if there's a change in pyUmbral.

---

This will still require the installation of new releases; however, perhaps it's more common to use `*`